src/pages/why-choose-us/_icon-grid.tsx

```diff
@@ -100,44 +101,44 @@ export const IconGrid = () => (
         <GridWrapper columns="repeat(3, 1fr)" column_gap="13rem" row_gap="10rem">
             <Col
                 Icon={ProvenTrackRecord}
-                title={localize('Proven track record')}
-                content={localize(
-                    'Deriv Group – the owners of the Deriv and Binary.com brands – has a history that dates back to 1999 when we laid the groundwork for the world’s first fixed-odds trading service. Since then, we have grown from strength to strength, garnering the trust of traders around the world.',
-                )}
+                title="_t_Proven track record_t_"
+                content={
+                    <Localize translate_text="_t_Deriv Group – the owners of the Deriv and Binary.com brands – has a history that dates back to 1999 when we laid the groundwork for the world’s first fixed-odds trading service. Since then, we have grown from strength to strength, garnering the trust of traders around the world._t_" />
```

Comment:
```suggestion
                    "_t_Deriv Group – the owners of the Deriv and Binary.com brands – has a history that dates back to 1999 when we laid the groundwork for the world’s first fixed-odds trading service. Since then, we have grown from strength to strength, garnering the trust of traders around the world._t_"
```